docs/source/install.rst

```diff
@@ -266,10 +266,11 @@ Create a build directory and step into it::
     mkdir build
     cd build
 
-From the build directory you can now configure CMake and build the binaries::
+From the build directory you can now configure CMake, build and install the binaries::
 
     cmake ..
     cmake --build .
+    make install
```

Comment:
This only works for "Unix Makefiles" generators. The generic install step is actually:

    cmake --build . --target install

Reply:
Ah, I see. Any reason why the ``--target install`` step is omittted?

Reply:
No reason, so this is a welcome contribution! Just suggesting a modification to this PR so that it is more generic.

Reply:
Excellent! Thank you. PR modified accordingly.